From 9d933c3c6535c1c63291e3d35f4ada9135d422df Mon Sep 17 00:00:00 2001
From: Alyssa Ross <hi@alyssa.is>
Date: Mon, 11 Mar 2019 02:08:43 +0000
Subject: [PATCH] Allow test_period_of_repetition to be slow

Recent versions of hypothesis default to a 200ms timeout, which wasn't
enough for my Thinkpad X220 to run this test. I've increased the timeout
for this single test to hopefully a reasonable amount for older
hardware.

(cherry picked from commit 752beb3a32b59f54168816da531c9d2a387f9715)
---
 test/test_priority.py | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/test/test_priority.py b/test/test_priority.py
index c98a28d..013ce30 100644
--- a/test/test_priority.py
+++ b/test/test_priority.py
@@ -12,7 +12,7 @@ import itertools
 
 import pytest
 
-from hypothesis import given
+from hypothesis import given, settings
 from hypothesis.strategies import (
     integers, lists, tuples, sampled_from
 )
@@ -489,6 +489,7 @@ class TestPriorityTreeOutput(object):
     fairness and equidistribution.
     """
     @given(STREAMS_AND_WEIGHTS)
+    @settings(deadline=None)
     def test_period_of_repetition(self, streams_and_weights):
         """
         The period of repetition of a priority sequence is given by the sum of
-- 
2.19.2

